src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -1331,8 +1331,14 @@ public void setGameMode(@NotNull GameMode gameMode) {
             sendChangeGameStatePacket(ChangeGameStatePacket.Reason.CHANGE_GAMEMODE, gameMode.getId());
 
             PlayerInfoPacket infoPacket = new PlayerInfoPacket(PlayerInfoPacket.Action.UPDATE_GAMEMODE);
+            PlayerAbilitiesPacket abilitiesPacket = new PlayerAbilitiesPacket();
+            if (gameMode.equals(GameMode.CREATIVE)) {
+                abilitiesPacket.flags = 0b1101;
```

Comment:
I haven't looked, but I assume there are constants somewhere for this which could be used. If not they should be added. At minimum a comment here is needed to say which flags are being set/what they do.

Reply:
i made those constants, its in PlayerAbilitiesPacket